core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -379,6 +380,12 @@ class SparkContext(config: SparkConf) extends Logging {
     override protected def initialValue(): Properties = new Properties()
   }
 
+  private var _cachedArrowBatchServerPort: Option[Int] = None
+  private[spark] def cachedArrowBatchServerPort: Int = _cachedArrowBatchServerPort.get
+
+  private var _cachedArrowBatchServerSecret: Option[String] = None
```

Comment:
Do we need the above two `var`s? Seems we can easily get it from `_cachedArrowBatchServer`